src/dataExplorer/components/Results.tsx

```diff
@@ -34,15 +34,16 @@ import {PersistanceContext} from 'src/dataExplorer/context/persistance'
 // Types
 import {FluxResult} from 'src/types/flows'
 import {RemoteDataState, SimpleTableViewProperties} from 'src/types'
+import {LanguageType} from 'src/dataExplorer/components/resources'
 
 // Utils
 import {bytesFormatter} from 'src/shared/copy/notifications'
 
 import './Results.scss'
-import {LanguageType} from './resources'
 
 const QueryStat: FC = () => {
   const {result} = useContext(ResultsContext)
+  const {resource} = useContext(PersistanceContext)
 
```

Comment:
Super minor (and could go in the next PR), but think this should be `PersistenceContext`.

Reply:
You got it! It's tracked in https://github.com/influxdata/ui/issues/6649 and I should have sometime this week to refactor it

Reply:
Thanks!